src/__tests__/jest/actions/socialLoginAction.test.js

```diff
@@ -0,0 +1,45 @@
+import moxios from "moxios";
+import thunk from "redux-thunk";
+import configureMockStore from "redux-mock-store";
+
+import { socialLoginSuccess } from "../../../redux/actions/SocialLoginAction";
+import * as actionTypes from "../../../redux/action_types";
+import { socialPayload } from "../../mock/mockData";
+
+const middleware = [thunk];
+const mockStore = configureMockStore(middleware);
+const loginURL =
+  "https://ah-magnificent6-staging.herokuapp.com/api/users/oauth/";
```

Comment:
Use the config file and .env to set up the BASE_URL

Reply:
@Bileonaire This has been fixed.